src/vivarium_census_prl_synth_pop/tools/jobmon.py

```diff
@@ -80,18 +82,50 @@ def run_make_results_workflow(
     )
 
     # Create tasks
-    task_make_results = template_make_results.create_task(
-        name="make_results_task",
-        upstream_tasks=[],
-        raw_output_dir=raw_output_dir,
-        final_output_dir=final_output_dir,
-        verbose=verbose_arg,
-        mark_best=mark_best_arg,
-        test_run=test_run_arg,
-        seed=seed_arg,
-        artifact_path=artifact_path,
-    )
-    workflow.add_task(task_make_results)
+    if seed_arg == "":  # Run all seeds in parallel
+        # All raw results are in the format <raw_output_dir>/<observer>/<observer>_<seed>.csv.bz2
+        seeds = sorted(
+            list(
+                set(
+                    [
+                        x.name.split(".")[0].split("_")[-1]
```

Comment:
Usually I'd say you could use `x.stem` instead of `x.name`, but unfortunately `Path("foo.csv.bz2").stem` is "foo.csv" not "foo".